koku/masu/processor/tasks.py

```diff
@@ -148,6 +160,11 @@ def get_report_files(
                 raise processing_error
 
     WorkerCache().remove_task_from_cache(cache_key)
+    if start_date:
+        start_date_str = start_date.strftime("%Y-%m-%d")
+        convert_to_parquet.delay(
```

Comment:
Similar question from the listener flow.  Should we call this after summary or should it be called for every file?  I'm happy to sort it out when I merge the worker changes, just want to understand if after summary it can be done or not.

Reply:
Our end goal would be to summarize data from the converted parquet files. Ideally this wouldn't happen for every file as it will convert each file in the manifest to parquet.

However, for now I think it could be done after summary.

Reply:
Yes, sounds like it could be a parallel task to our summary task.  Then one day we can just turn off our summary task or call the new summary_from_parquet logic once all manifest file line item is complete.

Reply:
I moved this call to work/look like the `summarize_manifest` task that gets called.